### ISSUE TITLE
NemosMiner v3.8.0.0 and beyond

### ISSUE THREAD

**Issue by Minerx117** [nemosminer/NemosMiner#976]
NemosMiner v3.8.0.0 and future Versions is now under a new Repository, as my Phone broke on me and i had 2FA setup and had not saved my ssh keys.

NemosMiner is being updated,  and can be found here: https://github.com/Minerx117/NemosMiner

Best Regards
Nemo/Minerx117

**Comment by grantemsley** [comment]
May want to update the links on nemosminer.com to match.

**Comment by grantemsley** [comment]
@MrPlusGH is this repo staying open and being updated now?  Or is @Minerx117 's version the new official one?  I thought this issue was being left open to direct people there now?

**Comment by Minerx117** [comment]
this Repo is no longer being Updated 

@Minerx117  is now the new official Repo for Nemosminer

this Issue was being left open to direct people there, but @MrPlusGH has decided to close it.

I have lost my phone and no longer have access to this repo only @MrPlusGH has access to it as i trusted him to help me work on it..

**Comment by Minerx117** [comment]
any commits to this Repo after May 7 are not from me, after May 7 i forked repo to: https://github.com/Minerx117/NemosMiner and have continued maintaining NemosMiner there 

Best Regards
Nemo/Minerx117

**Comment by grantemsley** [comment]
@Minerx117 do you still have control over nemosminer.com or did you lose access to that was well?

**Comment by Minerx117** [comment]
@grantemsley yes i have control over my digital ocean Account in which the nemosminer.com server is held so can close server at any time.
